webservices/templates/swagger-ui.html

```diff
@@ -30,16 +30,102 @@
+     // Provide a URL or e-mail address to be used for customer support.
+     //
+     // The format for e-mail addresses can be given as either
+     // 'example@example.com' or 'mailto:example@example.com'.
+     contactUrl: 'mailto:APIinfo@fec.gov',
+
+     // Provide the name of your developer site. This will appear in the
+     // subject of the welcome e-mail as "Your {{siteName}} API key".
+     siteName: 'openFEC',
+
+     // Provide a sender name for who the welcome email appears from. The
+     // actual address will be "noreply@api.data.gov", but this will
+     // change the name of the displayed sender in this fashion:
+     // "{{emailFromName}} <noreply@api.data.gov>".
+     emailFromName: 'FEC',
+
+     // Provide an example URL you want to show to users after they signup.
+     // This can be any API endpoint on your server, and you can use the
+     // special {{api_key}} variable to automatically substitute in the API
+     // key the user just signed up for.
+     exampleApiUrl: 'https://api.open.fec.gov/v1/legal/search/?api_key={{api_key}}',
+
+     // OPTIONAL: Provide extra content to display on the signup confirmation
+     // page. This will be displayed below the user's API key and the example
+     // API URL are shown. HTML is allowed. Defaults to ""
+     // signupConfirmationMessage: '',
+
+     // OPTIONAL: Set to true to verify the user's e-mail address by only
+     // sending them their API key via e-mail, and not displaying it on the
+     // signup confirmation web page. Defaults to false.
+     verifyEmail: true,
+
+     // OPTIONAL: Set to false to disable sending a welcome e-mail to the
+     // user after signing up. Defaults to true.
+     sendWelcomeEmail: true,
+
+     // OPTIONAL: Provide an extra input field to ask for the user's website.
+     // Defaults to false.
+     // websiteInput: true,
+
+     // OPTIONAL: Provide an extra checkbox asking the user to agree to terms
+     // and conditions before signing up. Defaults to false.
+     // termsCheckbox: true,
+
+     // OPTIONAL: If the terms & conditions checkbox is enabled, link to this
+     // URL for your API's terms & conditions. Defaults to "".
+     // termsUrl: "https://agency.gov/api-terms/",
+   };
+ </script>
+ <noscript>Please enable JavaScript to signup for an <a href="http://api.data.gov/">api.data.gov</a> API key.</noscript>
+
+    <script src="{{ swagger_static('dist/swagger-layout.js') }}">
+</script>
+
+{% if api_key_signup_feature_flag %}
+
     <script>
       window.onload = function() {
         SwaggerLayout(
           "{{ specs_url }}",
           document.getElementById("swagger-ui")
         );
+        
+         var apiUmbrella = document.createElement('script'); 
+         apiUmbrella.type = 'text/javascript'; 
+         apiUmbrella.async = true;
+         apiUmbrella.src = 'https://api.data.gov/static/javascripts/signup_embed.js';
+         (document.getElementsByTagName('head')[0] || 
```

Comment:
We can remove 
`|| document.getElementsByTagName('body')[0]`
Since we know we have a head in this document

Reply:
removed under issue https://github.com/fecgov/openFEC/pull/4359#pullrequestreview-421086744